ci/scripts/build_fabric.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+# Copyright the Hyperledger Fabric contributors. All rights reserved.
+#
+# SPDX-License-Identifier: Apache-2.0
+set -euo pipefail
+
+git clone -b master https://github.com/hyperledger/fabric "${GOPATH}/src/github.com/hyperledger/fabric"
+cd "${GOPATH}/src/github.com/hyperledger/fabric"
+make docker
+make release/linux-amd64
+cp release/linux-amd64/bin/* "${GOBIN}"
+cp sampleconfig/*yaml "${GOPATH}/src/github.com/hyperledger/fabric-samples/config"
```

Comment:
you cloned fabric repo in line 7, but not fabric-samples repo. This could fail if fabric-samples/config does not exist. If this is necessary to copy over there, it is not clear here. How about we clone the fabric-samples repo too (possibly conditionally if does not exist).

Reply:
The cloning of samples and explicit creation of that directory are part of the pipeline, you can see them `azure-pipeline-nightly.yml`. I'm actually more inclined to move the copy out of the build fabric script, since it's not actually part of building fabric. I've created a setup_fabric_samples script that performs this functionality, including the explicit creation of config and the copying